allensdk/brain_observatory/ecephys/align_timestamps/__main__.py

```diff
@@ -63,6 +63,7 @@ def align_timestamps(args):
         mapped_files = {}
 
         for timestamp_file in probe["mappable_timestamp_files"]:
+            print(timestamp_file["name"])
```

Comment:
this does not seem related. Please remove and open an issue to improve logging in the ecephys timestamps alignment module.